ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml

```diff
@@ -13,70 +13,14 @@ build_root:
     tag: latest
 canonical_go_repository: github.com/openshift/openshift-azure
 images:
-- dockerfile_path: images/sync/Dockerfile
+- dockerfile_path: images/azure/Dockerfile
```

Comment:
```suggestion
- dockerfile_path: images/arho/Dockerfile
```

Reply:
I'm unfamiliar with the namespace arho.  What/why this change?

Reply:
@kwoodson this combines with https://github.com/openshift/openshift-azure/pull/1464 to reduce the number of images we ship to simplify the release train